code/processes/rdb.q

```diff
@@ -190,10 +191,16 @@ reload:.rdb.reload
 /-load the sort csv
 .sort.getsortcsv[.rdb.sortcsv]
 
+if[.rdb.nulltickerplant;
+    // removes tp from connection list if in nulltickerplant mode
+    .servers.CONNECTIONS:.servers.CONNECTIONS except .rdb.tickerplanttypes;
+    .rdb.tplogdate:.z.d; // defines tplogdate for setpartition
+    ]
+
 .lg.o[`init;"searching for servers"];
 
 //check if tickerplant is available and if not exit with error 
-.servers.startupdepcycles[.rdb.tickerplanttypes;.rdb.tpconnsleepintv;.rdb.tpcheckcycles]
+.servers.startupdepcycles[.servers.CONNECTIONS;.rdb.tpconnsleepintv;.rdb.tpcheckcycles]
```

Comment:
Instead of doing it like this, could we have a variable called .rdb.connectonstart, which defaults to true. If true, it creates the connections using startupdepcycles and subscribes. In both cases I think it probably needs to set the partition.  So similar to what you have, but I think having a variable called nulltickerplant and then removing the tp from the connection list is a bit confusing.